packages/marble-api/openapis/marblecore-api.yaml

```diff
@@ -4,69 +4,69 @@ info:
   title: 'Marble Core API'
   description: 'Consume Marble internal endpoints'
 tags:
-- name: Admin
-  description: Admin operations.
-- name: Data Model
-  description: Operations about data model.
-- name: Authorization
-  description: Operations about authorization.
-- name: Analytics
-  description: Operations about analytics.
-- name: ApiKeys
-  description: Operations about api keys.
-- name: Users
-  description: Operations about users.
-- name: Organizations
-  description: Operations about organizations.
-- name: Decisions
-  description: Operations about decisions.
-- name: Cases
-  description: Operations about cases.
-- name: Ingestions
-  description: Operations about data ingestion.
-- name: Scenarios
-  description: Operations about scenarios.
-- name: Scenario Iterations
-  description: Operations about scenario iterations.
-- name: Scenario Iteration Rules
-  description: Operations about scenario iteration rules.
-- name: Scenario Publications
-  description: Operations about scenario publications.
-- name: CustomLists
-  description: Operations about custom lists.
-- name: Editor
-  description: Operations about editor.
-- name: Tags
-  description: Operations about tags.
-- name: Webhooks
-  description: Operations about webhooks.
-- name: RuleSnoozes
-  description: Operations about rule snoozes.
-- name: TestRun
-  description: Operations about testrun.
+  - name: Admin
```

Comment:
On my side, yaml is formatted using Prettier, and this doesn't introduce the large diff with withespace you have here.